frontend/package.json

```diff
@@ -93,7 +93,7 @@
     "openshift-logos-icon": "1.7.1",
     "patternfly": "^3.59.1",
     "patternfly-react": "2.32.3",
-    "patternfly-react-extensions": "2.18.4",
+    "patternfly-react-extensions": "3.0.2",
```

Comment:
This gives me pause since it is a major version bump.  All good?

Reply:
We need this commit for the `icon` prop: https://github.com/patternfly/patternfly-react/commit/694cd049ee324e07b32333be5406967425002fc8

We could potentially go to any version 2.20+. I picked the one we were using in 4.3.

It looks like the major version bump was for xterm 4.x. It doesn't seem to impact the console xterm version, though.

https://github.com/patternfly/patternfly-react/releases/tag/patternfly-react-extensions%403.0.0